test/Scripts.Integration.Test/common.ps1

```diff
@@ -276,13 +278,15 @@ function CheckSymbolServerOutput([string] $buildMethod, [string] $symbolServerOu
             if ($unity2020OrHigher)
             {
                 $expectedFiles = @(
-                    "libil2cpp.sym.so: count=$($withSources ? 3 : 2)"
+                    "libil2cpp.sym.so: count=$($withSources ? 2 : 1)",
+                    "libil2cpp.dbg.so: count=$($withSources ? 3 : 2)"
```

Comment:
won't this break 2021? THat worked before this change

Reply:
2021 is completely special cased on line 251

Reply:
right...